etc/config/checkstyle-verify.xml

```diff
@@ -60,13 +63,16 @@
             <property name="allowSingleLineStatement" value="true"/>
         </module>
         <module name="LeftCurly">
-            <property name="maxLineLength" value="130"/>
+            <property name="tokens" value="ANNOTATION_DEF, CLASS_DEF, CTOR_DEF, ENUM_CONSTANT_DEF, ENUM_DEF,
+                                           INTERFACE_DEF, LITERAL_CASE, LITERAL_CATCH, LITERAL_DEFAULT,
+                                           LITERAL_DO, LITERAL_ELSE, LITERAL_FINALLY, LITERAL_FOR, LITERAL_IF,
+                                           LITERAL_SWITCH, LITERAL_SYNCHRONIZED, LITERAL_TRY, LITERAL_WHILE,
+                                           METHOD_DEF, OBJBLOCK, STATIC_INIT" />
```

Comment:
This is default, if you were up to removing max line length, then just remove that property.

Reply:
LAMBDA is removed, added a comment